### ISSUE TITLE
Applying parallel attn with ff to existing pretrained model? 

### ISSUE THREAD

**Issue by huu4ontocord** [lucidrains/flamingo-pytorch#12]
Hi - awesome work! I am trying to understand ? I couldn't find a paper - only a reference to https://github.com/kingoflolz/mesh-transformer-jax. Is this right? Am I understanding that it is bascially applying multiple operations of for qkv and ff at once? Is it possible to use this trick to modify an existing pretrained model?

https://github.com/lucidrains/flamingo-pytorch/blob/749f8244794002371913d2fc4e7411afd5eddc67/flamingo_pytorch/flamingo_palm.py#L90

Many thanks in advance!

Huu

**Comment by lucidrains** [comment]
@ontocord yup that's correct, it was invented by Ben Wang for GPT-J, then subsequently adopted by [PaLM](https://arxiv.org/abs/2204.02311)
